CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v7.1.0
+------------------------------
+*January 22, 2018*
+
+### Changed
+- Added `useFozziePackageVersion` to config file to handle the css versioning, default value is set to `false`.
```

Comment:
Should make these variables more generic – `usePackageName` for example, so that it can be applied to any package that wants to include the package version in the CSS/JS builds.